api/WebApi.ts

```diff
@@ -318,7 +319,14 @@ export class WebApi {
         return new testm.TestApi(serverUrl, handlers, this.options);
     }
 
-	public async getTestResultsApi(serverUrl?: string, handlers?: VsoBaseInterfaces.IRequestHandler[]): Promise<testresultsm.ITestResultsApi> {
+    public async getTestPlanApi(serverUrl?: string, handlers?: VsoBaseInterfaces.IRequestHandler[]): Promise<testresultsm.ITestPlanApi> {
+        // TODO: Load RESOURCE_AREA_ID correctly.
+        serverUrl = await this._getResourceAreaUrl(serverUrl || this.serverUrl, "e4c27205-9d23-4c98-b958-d798bc3f9cd4");
```

Comment:
can we store this GUID in some constants file and then use it here? Since here we are hardcoding it

Reply:
Won't fix: This is area resource id which will never change, also this is how we are referencing for other areas as well.